tests/conftest.py

```
@@ -1,7 +1,55 @@
+def redis_mock(mocker):
+    """Mock Redis for testing."""
+    mock = mocker.patch('redis.Redis')
+    return mock
 
 @pytest.fixture
-def client():
-    return TestClient(app)
+def test_client(test_app):
+    """Create a test client."""
+    from fastapi.testclient import TestClient
+    return TestClient(test_app)
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)